deployments/cluster/educator.nix

```diff
@@ -0,0 +1,81 @@
+env: n: { lib, name, nodes, pkgs, resources, ... }: with lib;
+
+let
+  address = ip: ip + ":4010:4011";
+  inherit (nodes.witness-load-balancer.config.networking) publicIPv4;
+  isWitness = node: elem "witness" node.config.system.nixos.tags;
+in
+
+{
+  ##
+  # `map` to make additional SGs easier to add and SG list more readable
+  deployment.ec2.securityGroupIds = map (x: resources.ec2SecurityGroups."cluster-${x}-sg".name ) (
+    [ "educator-api-private" "witness-api-private" ]
+  );
+
+  networking.firewall.allowedTCPPorts = [
+    4040 4041   # Witness ZMQ API
+    4030        # Witness HTTP Wallet API
+    4040        # Educator HTTP API
+  ];
+
+  ##
+  # For each secret data point:
+  # * Add its file to deployment.keys
+  # * Add its name to services.disciplina-witness.keyFiles
+  # * Wrap its value in `services.disciplina-witness.args` with `cat`
+  deployment.keys = {
+    "witness-comm-sec".keyFile = ../../keys + "/${env}/witness/comm-sec";
+  };
+
+  services.disciplina = rec {
```

Comment:
I remember the good old days when we could specify multiple dscp nodes with the module. Maybe we're gonna need that in the future?

Reply:
Maybe. I just did the simplest thing possible.